app/views/episode_transcripts/_form_main.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form.fields_for :transcript, episode.transcript, include_id: false do |fields| %>
+  <div class="col-12 mb-4">
+    <div class="form-floating input-group">
+      <%= fields.select :format, episode_transcript_options, {} %>
```

Comment:
So for an episode

Reply:
resolved in `_form_transcript` below, since that is where a Transcript is first built (if new).